cmd/commands/cli/command.go

```diff
@@ -27,16 +27,18 @@ import (
 	"strings"
 	"time"
 
+	"github.com/mysteriumnetwork/node/cmd/commands"
+
+	remote_config "github.com/mysteriumnetwork/node/config/remote"
```

Comment:
Lets not give packages names if we dont have to. If a package requires a rename, we should change the actual name while/if we can.